CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to WoltModalSheet
+
+To get started with contributing, please follow the steps below:
+
+1. [Fork the wolt_modal_sheet repo](https://github.com/woltapp/wolt_modal_sheet/fork) on GitHub.
+2. Clone your forked repo locally.
+3. Ensure you have [Melos](https://pub.dev/packages/melos/install) installed.
+   ```bash
+   dart pub global activate melos
+   ```
+4. Use Melos to bootstrap the project.
+   ```bash
+   melos bootstrap
+   ```
+5. Create a new branch from the `main` branch.
+6. Make your changes.
+7. Create a pull request.
```

Comment:
We need to add a note here:
Before creating a pull request, run the Dart format and Flutter analyze for a merge check using Melos, ensuring compatibility with the minimum supported Flutter version, which is 3.7.2.
```yaml
melos run format
melos run analyze
```

Reply:
I thought about it. I think, it is better to include it in the pull request template. So that they will need to click check mark to verify that melos analyze doesn't produce any errors, etc.

Reply:
So in contributing.md; we introduce how the contribution works, and handle the details in the respective area, in this case in the PR.